Content.Server/GameTicking/Rules/ThiefRuleSystem.cs

```diff
@@ -125,8 +122,10 @@ public bool MakeThief(ICommonSession thief, bool AddPacified)
         });
 
         //Add Pacified
-        if (AddPacified)
+        if (addPacified) //This check is important because some servers may want to disable the thief's pacifism. Do not remove.
+        {
             EnsureComp<PacifiedComponent>(mind.OwnedEntity.Value);
+        }
 
```

Comment:
Long-term this should just be using the antag code to add components.

Reply:
marked as To Do